spec/helpers/granule_filter_matchers.rb

```diff
@@ -2,8 +2,8 @@
   match do |selector|
     wait_for_xhr
     synchronize do
-      number_granules = expect(selector.text).to match_regex /\d+ Granule/
-      after_granule_count = number_granules.to_s.split(" ")[0].to_i
+      number_granules = expect(selector.text).to match_regex(/\d+ Granule/)
+      after_granule_count = number_granules.to_s.split(' ')[0].to_i
```

Comment:
I think a better way to do this would be to use something like `/(\d+) Granule/` to have the response of the regex be the number instead of manipulating the response.